core/src/main/resources/hudson/model/UsageStatistics/global.groovy

```diff
@@ -1,7 +1,19 @@
 package hudson.model.UsageStatistics
 
+import hudson.model.UsageStatistics
+import jenkins.security.FIPS140
+import hudson.Functions
+
 def f=namespace(lib.FormTagLib)
 
 f.section(title: _("Usage Statistics")) {
-    f.optionalBlock(field: "usageStatisticsCollected", checked: app.usageStatisticsCollected, title: _("statsBlurb"))
+    if (UsageStatistics.DISABLED) {
+        span(class: "jenkins-not-applicable") {
+            raw(_("disabledBySystemProperty"))
+        }
+    } else if (FIPS140.useCompliantAlgorithms()) {
+        f.optionalBlock(field: "usageStatisticsCollected", checked: app.usageStatisticsCollected, title: _("statsBlurbFIPS"))
```

Comment:
It might be useful to be explicit here, that general usage statistics are not sent, e.g. in a `f:description`.